presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -80,6 +83,9 @@
     private final Map<String, DockerContainer> containers;
     private final Optional<EnvironmentListener> listener;
 
+    private static final ExecutorService executorService = Executors
+            .newCachedThreadPool(new ThreadFactoryBuilder().setNameFormat("environment-%d").build());
```

Comment:
io.airlift.concurrent.Threads#daemonThreadsNamed?

Reply:
Didn't know that :) What's the difference?